migtests/lib/yb.py

```diff
@@ -149,3 +149,14 @@ def invalid_index_present(self, table_name, schema_name):
 				return True
 
 		return False
+
+
+	def fetch_all_triggers(self, schema_name="public") -> List[str]:
```

Comment:
Rename to `list_trigger_names(...)` and move it immediately after the `get_table_names(...)`. (You should also consider renaming the `get_table_names()` as `list_table_names()`.